pynucastro/networks/tests/test_derived_network.py

```diff
@@ -67,11 +67,7 @@ def fe52__p_mn51__derived(tf):
     rate += np.exp(  {:.14f} + -85.6326403498911*tf.T9i + -36.1825*tf.T913i + 0.873042*tf.T913
                   + -2.89731*tf.T9 + 0.364394*tf.T953 + 0.833333*tf.lnT9)
 
-    p_temp_array = np.array([0.01, 0.15, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0, 1.5, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5.0, 6.0, 7.0, 8.0, 9.0, 10.0])
-    p_pf_array = np.array([1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0])
-    p_pf_exponent = np.interp(tf.T9, xp=p_temp_array, fp=np.log10(p_pf_array))
-    p_pf = 10.0**p_pf_exponent
-
+    p_pf = 1.0
```

Comment:
can you add a comment here -- what is `p_pf`?

Reply:
is the partition function of the proton, but yes I can put an extra comment

Reply:
but rate is not a string right?  where is this substituted?

Reply:
I guess with the new modifications I've made; everything gets clearer. The point is to take the reverse coefficients and compute the final rate, by `G_{nucr1} * G_{nucr2} .../ G_{nucp1} * G_{nucp2} * ...` If the partition function of the nuclei is not known, them we set it up to 1.0.

Reply:
The .join method will construct the * string in the numerator and denominator